e2e/tests/post-details.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { test, expect } from '@playwright/test';
+import { $path } from 'remix-routes';
+
+import { formatPageHeaderDate } from '../../app/components/ds/page/PageHeader';
+import { getPosts } from '../../app/services/Content/posts';
+
+test.describe('PostList', async () => {
+  getPosts().forEach((post) =>
+    test(
+      `Test page: ${post._meta.id}`,
+      {
+        annotation: [
+          {
+            type: 'environment',
+            description: 'dev',
+          },
+        ],
```

Comment:
I wanted to experiment with this to see how it could be used as metadata when deciding which tests to run in CI.

Based on some inferred information in the PR, build a list of related jira ticket ids and feed that into a playwright test filter.

so: 

- infer jira tickets from PR (branch name, comments)
  - based on those, look for linked story tickets
- take the resulting flat list of jira ids and tell playwright to only run tests which are annotated with these ticket ids.